.golangci.yml

```
@@ -0,0 +1,110 @@
+    - nolintlint
+    - rowserrcheck
+    - typecheck
+    - unconvert
+    - unparam
+    - whitespace
+  
+  settings:
+    errcheck:
+      check-type-assertions: true
+      check-blank: true
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)